services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -211,19 +217,26 @@ const CMSUserRoutes = ({
 
                 <Route element={<SubmissionSideNav />}>
                     <Route
-                        path={RoutesRecord.SUBMISSIONS_QUESTIONS_AND_ANSWERS}
+                        path={RoutesRecord.SUBMISSIONS_CONTRACT_QUESTIONS_AND_ANSWERS}
                         element={<QuestionResponse />}
                     />
                     <Route
-                        path={RoutesRecord.SUBMISSIONS_UPLOAD_QUESTION}
-                        element={<UploadQuestions />}
+                        path={RoutesRecord.SUBMISSIONS_UPLOAD_CONTRACT_QUESTION}
+                        element={<UploadContractQuestions />}
                     />
                     <Route
                         path={RoutesRecord.SUBMISSIONS_SUMMARY}
                         element={<SubmissionSummary />}
                     />
                 </Route>
 
+                {showQAByRates &&
+                    <Route
```

Comment:
Depending on who's PR merges first, this will have to be moved into the rate summary sidenav layout route I created in this [PR](https://github.com/Enterprise-CMCS/managed-care-review/pull/2847/files#diff-7c6dba41ba307e3f1776538a7c8ad37bff196c9bb4059cb92c124ee34ca29757). 

So if my PR merges first then moving this would be done in this PR.

Reply:
@JasonLin0991 Is there going to be a rate summary sidenav layout on state user though?

Reply:
Currently states do not see single rate summaries so no need for a sidenav layout for rates.